src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
+$main-accent: #060b35;
+$secondary-accent: #616070;
+$blue-accent: #00acdc;
+
+@mixin set-font($size, $weight, $height, $color) {
+  font-size: $size;
+  font-weight: $weight;
+  line-height: $height;
+  color: $color;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+
+  display: flex;
```

Comment:
This property redundant here as by default block elements takes 100% of available width